SignalServiceKit/src/Storage/TSDatabaseView.m

```diff
@@ -295,30 +300,26 @@ + (YapDatabaseViewSorting *)messagesSorting {
 
 + (void)asyncRegisterSecondaryDevicesDatabaseView:(OWSStorage *)storage
 {
-    YapDatabaseViewGrouping *viewGrouping =
-        [YapDatabaseViewGrouping withObjectBlock:^NSString *_Nullable(YapDatabaseReadTransaction *_Nonnull transaction,
-            NSString *_Nonnull collection,
-            NSString *_Nonnull key,
-            id _Nonnull object) {
-            if ([object isKindOfClass:[OWSDevice class]]) {
-                OWSDevice *device = (OWSDevice *)object;
-                if (![device isPrimaryDevice]) {
-                    return TSSecondaryDevicesGroup;
-                }
+    YapDatabaseViewGrouping *viewGrouping = [YapDatabaseViewGrouping withObjectBlock:^NSString *_Nullable(
+        YapDatabaseReadTransaction *transaction, NSString *collection, NSString *key, id object) {
+        if ([object isKindOfClass:[OWSDevice class]]) {
+            OWSDevice *device = (OWSDevice *)object;
+            if (![device isPrimaryDevice]) {
+                return TSSecondaryDevicesGroup;
```

Comment:
I'm assuming this is clang-format gone wild.

Reply:
Seems that way.